euphoria-operator-testkit/src/main/java/cz/seznam/euphoria/operator/test/ReduceByKeyTest.java

```diff
@@ -86,7 +88,7 @@ public void testReductionType0() {
         return ReduceByKey.of(input)
             .keyBy(e -> e % 2)
             .valueBy(e -> e)
-            .reduceBy(Sets::newHashSet)
+            .reduceBy((ReduceFunction<Integer, HashSet<Integer>>) Sets::newHashSet)
```

Comment:
Yeah, there is no way around this (that I can think of). But I don't think that this will make any real user-facing problem, most of the time, the lambda will be correctly disambiguated.